core/backend/src/workspace/SettingsSchemas.ts

```diff
@@ -51,35 +55,125 @@ export interface SettingSchemaGroup {
 export class SettingsSchemas {
   private constructor() { } // singleton
```

Comment:
Singleton seems wrong. SettingsSchemas is a big ball of mutable state that persists after `IModelHost.shutdown`, seems like IModelHost ought to own it.

Reply:
I agree.